ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx

```diff
@@ -81,7 +83,7 @@ const LivePage = ({
       />
       <LinkedDataContainer type="CollectionPage" seoTitle="Test Live Page" />
       <main>
-        <Header title={title} description={description} />
+        <Header isLive={isLive} title={title} description={description} />
```

Comment:
Just thinking if at some stage this will get muddled with the `isLive` function we use for checking the environment? May be a non-issue really, as its rare we keep `isLive()` checks around for long.

Reply:
Yes I did wonder this. Alternatively, we could change the prop to be showLiveLabel (or something else).

Reply:
Yep it is the entire label. I have updated